src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -32,24 +33,29 @@
 
     private final String name;
     private final String phone;
+
+
```

Comment:
extra new lines, also some formatting issues with this file, where lines <120 chars are being broken up for no reason, leading to changes in random lines.

Reply:
resolved